dev-python/typer/typer-0.15.3.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+DISTUTILS_USE_PEP517=pdm-backend
+
+inherit distutils-r1 shell-completion pypi
+
+DESCRIPTION="Build great CLIs. Easy to code. Based on Python type hints."
+HOMEPAGE="
+	https://typer.tiangolo.com/
+	https://github.com/tiangolo/typer
+	https://pypi.org/project/typer/
+"
+SRC_URI="$(pypi_sdist_url "${PN}" "${PV}")"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+IUSE="+rich shell-detection test"
+
+BDEPEND="
+	test? (
+		$(python_gen_cond_dep '
```

Comment:
Why are you using `python_gen_cond_dep`?

Reply:
I am sorry, I did this without understand what I am copying from another ebuild for this python package.

Reply:
This is now removed.